### DISCUSSION TITLE
vim.keymap.set

### DISCUSSION THREAD

**Discussion by traxys** [nix-community/nixvim#94] [General]
I have a module that handles keybinds using the vim.keymap.set function, but I don't know if it's the best way to be done. 

If anybody wants to take a look: https://github.com/traxys/nvim-flake/blob/01822fde1fcd2c2f7ec735d4eef818429c9ef9fb/modules/core/default.nix#L66-L150

**Comment by pta2002** [comment]
Yeah, that's something I want to switch to for sure. I kind of need to completely rewrite the keybinds module anyway in order to better support things like setting keybinds from plugins specifically, was kind of just putting it off until 0.7 was in stable (see #15)
